['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpn_uig1ct/solution_0.py", line 67, in <module>\n    print(Solution().subsequencePairCount(nums = [2,4,6,8]))\nTypeError: subsequencePairCount() got an unexpected keyword argument \'nums\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpn_uig1ct/solution_1.py", line 67, in <module>\n    print(Solution().subsequencePairCount(nums = [1,2,3,4]))\nTypeError: subsequencePairCount() got an unexpected keyword argument \'nums\'\n']